Code/GraphMol/Descriptors/AtomicEnvironmentVector_catch.cpp

```diff
@@ -0,0 +1,292 @@
+//
+//  Copyright (C) 2020 Manan Goel
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include "RDGeneral/test.h"
+#include "catch.hpp"
+
+#include <RDGeneral/Invariant.h>
+#include <GraphMol/RDKitBase.h>
+#include <GraphMol/FileParsers/MolSupplier.h>
+#include <GraphMol/FileParsers/FileParsers.h>
+#include <RDGeneral/RDLog.h>
+#include <vector>
+#include <algorithm>
+#include <fstream>
+#include <string>
+#include <sstream>
+#include <iterator>
+#include <GraphMol/GraphMol.h>
+#include <GraphMol/MolOps.h>
+#include <boost/tokenizer.hpp>
+#include <GraphMol/PeriodicTable.h>
+#include <GraphMol/Descriptors/AtomicEnvironmentVector.h>
+#include <Eigen/Dense>
+
+using namespace Eigen;
+
+std::vector<std::string> tokenize(const std::string &s) {
+  boost::char_separator<char> sep(", \n\r\t");
+  boost::tokenizer<boost::char_separator<char>> tok(s, sep);
+  std::vector<std::string> tokens;
+  std::copy(tok.begin(), tok.end(),
+            std::back_inserter<std::vector<std::string>>(tokens));
+  return tokens;
+}
+
+TEST_CASE("Symmetry Function Accuracy", "[Symmetry Function]") {
+  SECTION("CH4") {
+    std::string pathName = getenv("RDBASE");
+
+    std::string molFile =
+        pathName + "/Code/GraphMol/Descriptors/test_data/CH4.mol";
+
+    std::string fNameSF =
```

Comment:
Q: Where did the reference data that you use in the functions in this file come from?

Reply:
CH4 : https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:16183
SO2 : https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:18422
NH3 : https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:16134
Ethanol : https://pubchem.ncbi.nlm.nih.gov/compound/Ethanol

Reply:
The data from all these was then given as input to torchANI and the output from that was stored in the *.out files to check against.

Reply:
Ok, thanks. I wanted to make sure that the data in the .out files came from torchANI